limine-mkconfig

```diff
@@ -1,12 +1,12 @@
 #!/bin/env sh
 
-exec >/dev/stderr
+exec >&2
 
 source /etc/os-release
 source /etc/limine.conf
 
 OPTSTRING=":o:b:"
-CFGOUT=/dev/stdout
+CFGOUT="&1"
```

Comment:
This seems to not work properly (even before the changes): output configuration will still be printed to stderr since the `exec` basically points stdout file to stderr. I'd suggest replacing all `echo` log below with `>&2 echo`.